Example/src/LightboxExample.js

```diff
@@ -38,7 +38,7 @@ const styles = StyleSheet.create({
   container: {
     paddingTop: 0,
     height:
-      Platform.OS === 'web' ? dimensions.height - Header.HEIGHT : undefined,
+      Platform.OS === 'web' ? dimensions.height - useHeaderHeight() : undefined,
```

Comment:
Will it work at top level?

Reply:
It's a hook so it is not possible. I am not sure if my current implementation is valid, but I don't know how to do it better atm.